apps/prairielearn/src/pages/publicAssessmentQuestionsPreview/publicAssessmentQuestionsPreview.ts

```diff
@@ -0,0 +1,70 @@
+import * as express from 'express';
+import asyncHandler from 'express-async-handler';
+import { z } from 'zod';
+
+import * as error from '@prairielearn/error';
+import { queryRow, loadSqlEquiv } from '@prairielearn/postgres';
+
+import { type Assessment, AssessmentSchema } from '../../lib/db-types.js';
+import { selectCourseById, selectCourseIdByInstanceId } from '../../models/course.js';
+import { selectAssessmentQuestions } from '../../models/questions.js';
+
+import { InstructorAssessmentQuestions } from './publicAssessmentQuestionsPreview.html.js';
+
+async function selectAssessmentById(assessment_id: string): Promise<Assessment> {
+  return await queryRow(
+    sql.select_assessment_by_id,
+    {
+      assessment_id,
+    },
+    AssessmentSchema,
+  );
+}
```

Comment:
This belongs in a `models/assessment.ts` file.

Reply:
Moved to new models/assessment.ts file